<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ACIModeWindow</class>
 <widget class="QMainWindow" name="ACIModeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>通信模式设置</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>通信设置</string>
    </property>
    <widget class="QPlainTextEdit" name="PortNumber">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="centerOnScroll">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>端口号</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>工作模式</string>
     </property>
     <widget class="QRadioButton" name="Workmode_Client">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>19</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>客户端</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="Workmode_Server">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>19</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>服务端</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QCommandLinkButton" name="BtnStart">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>启动通信系统</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
